src/app/modules/mathjax/math-content.directive.ts

```diff
@@ -0,0 +1,40 @@
+import { Directive, OnInit, OnChanges, OnDestroy, Input, ElementRef, SimpleChanges } from "@angular/core";
+import { Subject } from "rxjs";
+import { take, takeUntil } from 'rxjs/operators';
+import { MathService } from "./math.service";
+
+@Directive({
+    selector: '[mathContent]'
```

Comment:
You can highlight the element by clicking on it. I think we should remove the border or at least a transparent border should be there. What do you think?